openhtf/__init__.py

```diff
@@ -349,7 +349,7 @@ def create_arg_parser(add_help=False):
     # repeat_limit option, this will be treated as a STOP.
     'REPEAT',
     # Causes the framework to ignore the measurement outcomes and execute the
-    # next phase.
+    # next phase.  The phase is still logged, unlike with run_if.
```

Comment:
can you add to this comment that the phase is logged, but can not cause the test to fail?